test/unit/ResultsList.test.js

```diff
@@ -65,9 +66,11 @@ describe('Results List', () => {
     });
 
     describe('Complete Results Data', () => {
+      let resultsBlock;
       let resultsData;
       before(() => {
-        resultsData = component.instance().formatAllResultsData(results.data.works, 'eReaderUrl', 'Referrer')[0].props;
+        resultsBlock = component.instance().formatAllResultsData(results.data.works, 'eReaderUrl', 'Referrer')[0];
+        resultsData = resultsBlock.props.children[0].props;
```

Comment:
why did this change?  seems ... disconcerting.

Reply:
To put the citation with the card I wrapped them in an empty element, so this just follows that and gets the card as a child. Nothing changed beyond that so I think it's okay?

Reply:
Why is it affecting components that don't have citations in them?

Reply:
I thought it wasn't? I updated the `return` from `formatAllResultsData` that includes the citation if the flag is enabled, should I change that so if the flag isn't present it renders differently? This felt like the cleanest solution to me.

Reply:
hmmm maybe I'm reading this wrong.  I thought this test was for a result without citation, so resultsData should return the same format as before, but here it's nesting a level down, even without the citation?  I don't think it's a huge deal, just stood out to me.

Reply:
it does nest even without the citation. I put the `key` in a wrapping `div` so that when the citation is present React doesn't complain about not having a unique key for repeating elements. Maybe there's a more elegant way to handle this but I've stared at for a while now and can't figure out what that would be

Reply:
aaah I'm so sorry I see what's happening now.  This is fine.